rpa-recorder/6.4/modules/ROOT/pages/user-interface.adoc

```diff
@@ -15,6 +15,7 @@ The *Recording Toolbar* appears on the screen when RPA Recorder starts recording
 [%header,cols="10,20,30,10"]
 |===
 |Icon |Name | Description | Hotkey
+|image:image-based-mouse-clicks-icon.png[Image-Based Mouse Clicks icon, 20%, 20%]|Image-based mouse clicks | Adds a recording with the <<Image-Based Mouse Clicks>> feature. | `F3`
```

Comment:
suggestion: how about "Adds a the Image-Based Mouse Clicks to the current recording." ?